client/src/app/services/api/api.service.ts

```diff
@@ -1,9 +1,17 @@
 import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
 
 @Injectable({
   providedIn: 'root'
 })
 export class ApiService {
 
-  constructor() { }
+  baseUrl = 'http://localhost:8080/api/';
```

Comment:
About the baseUrl: since both images (backend and frontend) are deployed on the same host on docker, I think it should be fine if we keep it at localhost. The frontend will successfully fetch from the local service on port 8080.